src/ipwndfu/usbexec.py

```diff
@@ -126,10 +134,10 @@ def match(self, info):
     ),
 ]
 
-EXEC_MAGIC = b"execexec"
-DONE_MAGIC = b"donedone"
-MEMC_MAGIC = b"memcmemc"
-MEMS_MAGIC = b"memsmems"
+EXEC_MAGIC = b"execexec"[::-1]
```

Comment:
wouldn't "reverse()" be more useful?  or at least a comment indicating this is a change of endian?  (it moves the wrong way on readability)